### PR TITLE
remove csr v1beta1 usage in integration tests

### PR DESCRIPTION
**Pull request by deads2k**
Getting ready for 1.22 when v1beta1 won't be present.  Some tests are checking API defaulting behavior.  Those have been moved to a deprecated file so we can remove the entire file when master becomes 1.22.

/kind cleanup
/priority important-soon
@kubernetes/sig-auth-pr-reviews 

```release-note
NONE
```

### CODE REVIEW

File: test/integration/certificates/controller_approval_test.go [hunk 1]
```diff
@@ -49,43 +49,44 @@ func TestController_AutoApproval(t *testing.T) {
...
-			signerName:   certv1beta1.KubeAPIServerClientKubeletSignerName,
-			request:      pemWithGroup("system:notnodes"),
-			autoApproved: false,
-		},
+		// usages are now required in v1.
+		//"should not auto-approve CSR that has kube-apiserver-client-kubelet signerName that does not match requirements": {
+		//	signerName:   certv1.KubeAPIServerClientKubeletSignerName,
+		//	request:      pemWithGroup("system:notnodes"),
+		//	autoApproved: false,
+		//},
```
Comment (enj) [inline test/integration/certificates/controller_approval_test.go:87]: nit: did you forget to delete this in the second commit?

**Review by enj — APPROVED** [review]
One minor comment.

File: test/integration/certificates/controller_approval_test.go (hunk 1 above)
Comment (deads2k) [reply test/integration/certificates/controller_approval_test.go:87]: > nit: did you forget to delete this in the second commit?

I did. now removed.

**Pull request merged**